### DISCUSSION TITLE
plugin vuepress-plugin-social-share

### DISCUSSION THREAD

**Discussion by DjCodeGit** [vuepress-theme-hope/vuepress-theme-hope#1732] [Q&A] [answered]
Hello,

I would like use this plugin **vuepress-plugin-social-share** with your theme. How can declare it in the config.ts. 
who could help me?

thanks a lot

**Comment by Mister-Hope** [comment, marked as answer]
I don’t think that plugin will work, make sure you are using plugin designed for vuepress2 instead of vp1.

we provide add-this plugin for you to “share”, maybe you could try that.

also we do have plans to build a v2 share plugin at https://github.com/orgs/vuepress-theme-hope/projects/1
